Tetris.ino

```diff
@@ -312,6 +321,14 @@ void manageGame() {
   }
 }
 
+void drawBack2Back()
```

Comment:
same for regular tetris.
perhaps doing something with the background to signify this ?